vhdbuilder/scripts/linux/tool_installs.sh

```diff
@@ -69,3 +69,7 @@ disableSystemdIptables() {
     # Mask the iptables service to prevent it from ever re-enabling and breaking pod networking.
     systemctl mask iptables || exit $ERR_DISBALE_IPTABLES
 }
+
+enableCgroupV2forAzureLinux() {
+    sed -i 's/systemd.legacy_systemd_cgroup_controller=yes systemd.unified_cgroup_hierarchy=0//g' /boot/systemd.cfg
```

Comment:
hrm. what bootloader do you use? you don't need an `update-grub` or similar here? or is this already a drop in of some sort?

Reply:
We use grub2 as our bootloader. update-grub eventually calls grub-mkconfig and that utility is currently not supported in Azure Linux. Support of this feature is within our radar which will be addressed in the next gen of Azure Linux but before that happens, we'll have to manually update the boot config to turn on cgroupv2